packages/editor-ui/src/Interface.ts

```diff
@@ -951,11 +953,27 @@ export interface ViewItemProps {
 	title: string;
 	description: string;
 	icon: string;
-	tag?: string;
+	tag?: {
+		text: string;
+		type: (typeof ElTag)['type'];
+	};
+	borderless?: boolean;
 }
 export interface LabelItemProps {
 	key: string;
 }
+export interface LinkItemProps {
+	url: string;
+	key: string;
+	newTab?: boolean;
+	title: string;
+	description: string;
+	icon: string;
+	tag?: {
+		text: string;
+		type: (typeof ElTag)['type'];
+	};
```

Comment:
Seen this literal type for tag in a few places, could be nice to have a separate type here, `NodeCreatorTag`?